app/models/filter.rb

```diff
@@ -0,0 +1,15 @@
+class Filter < ApplicationRecord
+  belongs_to :user, required: true, class_name: 'User'
```

Comment:
You should be able to drop the `class_name` argument here - Rails should infer it from the `:user` name. `required` also isn't strictly necessary: any reference that isn't explicitly optional is required by default.